### PR TITLE
Overhaul

### PR DESCRIPTION
**Pull request by rcgottlieb**
## Changes proposed in this pull request:
- Removed the Postgres database requirement
- merged the requirements.in in and updated it
- cleaned up the code

## security considerations
None

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -130,7 +130,7 @@ jobs:
       IAM_KEYS_HOST: ((aws-iam-keys-host))
       IAM_KEYS_PORT: ((aws-iam-keys-port))
       IAM_CREATE_TABLES: ((aws-iam-create-tables-bool))
-      GATEWAY_HOST: prometheus-production.service.cf.internal
+      GATEWAY_HOST: prometheus-staging.service.cf.internal
```
Comment (markdboyd) [inline ci/pipeline.yml:133]: I assume this is not intentional and we want to send the updated metrics to production?

Comment (rcgottlieb) [reply ci/pipeline.yml:133]: Yep, that is correct. I'm doing more work on this and will probably remove this PR, but I will double check that I updated the GATEWAY on the next PR. Thanks!

File: ci/aws-iam-check-keys/requirements.txt [hunk 2]
```diff
@@ -4,28 +4,40 @@
 #
 #    pip-compile
 #
-boto3==1.34.134
+--trusted-host pypi.python.org
```
Comment (cweibel) [inline ci/aws-iam-check-keys/requirements.txt:7]: Not sure you want the --trusted-host settings here, Ben mentioned this on one of my commits

**Review by cweibel — APPROVED** [review]
Long .gitignore aside, looks fine to test in staging

**Pull request merged**